adapters/x509_openssl.c

```diff
@@ -75,7 +75,7 @@ static int load_certificate_chain(SSL_CTX* ssl_ctx, const char* certificate)
                 // certificates.
 
                 /* Codes_SRS_X509_OPENSSL_07_006: [ If successful x509_openssl_add_ecc_credentials shall to import each certificate in the cert chain. ] */
-#if (OPENSSL_VERSION_NUMBER >= 0x10100000L) && (OPENSSL_VERSION_NUMBER < 0x20000000L)
+#if (OPENSSL_VERSION_NUMBER >= 0x10100000L) && (OPENSSL_VERSION_NUMBER < 0x20000000L) || defined(LIBRESSL_VERSION_NUMBER)
```

Comment:
If LibreSSL is always compatible with OpenSSL > 1.1, there are several other places that would require updates outside of these 3 in tlsio_openssl.c:
- x509_openssl.c (2 places)
- x509_openssl_ut.c (2 places)

Reply:
OK, I'll update them.

Reply:
Thank you! I've enqueued a test run.